### PR TITLE
77730: Create IPO

### PR DESCRIPTION
**Pull request by bratli**
Create IPO

### CODE REVIEW

**Comment by bratli** [comment]
Need some help with migration scripts - some fields on invitation should be nullable: false

File: src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/CreateInvitation/CreateInvitationCommandValidatorTests.cs [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        private CreateInvitationCommandValidator _dut;
+        private Mock<IInvitationValidator> _invitationValidatorMock;
+        private CreateInvitationCommand _command;
+        private readonly string _projectName = "Project name";
+        private readonly string _title = "Test title";
+        private readonly string _body = "body";
+        private readonly string _location = "location A";
+        private readonly DisciplineType _type = DisciplineType.DP;
+
+        private readonly IList<McPkgScopeForCommand> _mcPkgScope = new List<McPkgScopeForCommand>
```
Comment (llykkelarsen) [inline src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/CreateInvitation/CreateInvitationCommandValidatorTests.cs:24]: unused

File: src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/EditInvitation/CreateInvitationCommandHandlerTests.cs [hunk 2]
```diff
@@ -19,6 +19,9 @@ public class EditInvitationCommandHandlerTests
         private Mock<IInvitationRepository> _invitationRepositoryMock;
 
         private readonly string _plant = "PCS$TEST_PLANT";
```
Comment (llykkelarsen) [inline src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/EditInvitation/CreateInvitationCommandHandlerTests.cs:21]: Wrong name in the solution explorer, should be EditInvitationCommandHandlerTests?

File: src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs [hunk 3]
```diff
@@ -1,4 +1,6 @@
-﻿using System.ComponentModel.DataAnnotations;
+﻿using System;
```
Comment (llykkelarsen) [inline src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs:1]: remove

**Review by llykkelarsen — CHANGES_REQUESTED** [review]

**Comment by llykkelarsen** [comment]
Looks like the build fails with message "Column names in each table must be unique. Column name 'ProjectName' in table 'Invitations' is specified more than once."  Did invitations table already have a ProjectName column?

File: src/Equinor.ProCoSys.IPO.WebApi/appsettings.json [hunk 4]
```diff
@@ -9,11 +9,6 @@
   "ConnectionStrings": {
     "IPOContext": ""
   },
-  "Meetings": {
```
Comment (bratli) [inline src/Equinor.ProCoSys.IPO.WebApi/appsettings.json:12]: @llykkelarsen, should these changes just be local?

File: src/Equinor.ProCoSys.IPO.WebApi/Properties/serviceDependencies.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "secrets1": {
+      "type": "secrets"
```
[the remark this answers is not in the document]
Comment (bratli) [reply src/Equinor.ProCoSys.IPO.WebApi/Properties/serviceDependencies.json]: Sorry, this is just a local change that always appears, so committed it by accident :(

File: src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/CreateInvitationDtoValidator.cs [hunk 6]
```diff
@@ -4,19 +4,13 @@ namespace Equinor.ProCoSys.IPO.WebApi.Controllers.Invitation
 {
     public class CreateInvitationDtoValidator : AbstractValidator<CreateInvitationDto>
     {
-        public CreateInvitationDtoValidator() => RuleFor(x => x.Meeting).NotNull().SetValidator(new CreateMeetingDtoValidator());
-    }
-
-    public class CreateMeetingDtoValidator : AbstractValidator<CreateMeetingDto>
-    {
-        public CreateMeetingDtoValidator()
+        public CreateInvitationDtoValidator()
```
[the remark this answers is not in the document]
Comment (bratli) [reply src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/CreateInvitationDtoValidator.cs:7]: Jan Inge confirmed this Lykke, you are correct

File: src/Equinor.ProCoSys.IPO.WebApi/appsettings.json (hunk 4 above)
Comment (llykkelarsen) [reply src/Equinor.ProCoSys.IPO.WebApi/appsettings.json:12]: no pretty sure they should be checked in

**Review by llykkelarsen — APPROVED** [review]

**Pull request merged**